StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkURLGenerator.swift

```diff
@@ -50,7 +50,7 @@ struct LinkURLParams: Encodable {
     var intentMode: IntentMode
     var setupFutureUsage: Bool
     var cardBrandChoice: CardBrandChoiceInfo?
-    var linkFundingSources: Set<LinkSettings.FundingSource>
+    var linkFundingSources: [LinkSettings.FundingSource]
```

Comment:
The idea behind using a set was to not care about the ordering in this test (and in general), but I guess the custom `Equatable` implementation doesn’t really allow for that 😔

Reply:
The issue is that we're serializing it to JSON data, and we sort the keys when doing that, but if we use a `Set` then the ordering is non-deterministic during the test.